### PR TITLE
Mermaid statue scale bugfix

### PR DESCRIPTION
**Pull request by EpochFlame**
properly remove the mermaid scale from inventory when placing into the mermaid statue
also annotated a value and fixed minor typos in some comments

### CODE REVIEW

File: patches/tradeSequence.py [hunk 1]
```diff
@@ -267,14 +267,15 @@ def patchMermaid(rom):
 
 def patchMermaidStatue(rom):
     rom.patch(0x18, 0x095D, 0x0962, "", fill_nop=True)
-    rom.patch(0x18, 0x0966, 0x097A, ASM("""
+    rom.patch(0x18, 0x0966, 0x0971, ASM("""
```
Comment (daid) [inline patches/tradeSequence.py:270]: 0971 would end in the middle of an `ld a, XX` instruction (end addresses for these types of patches are excluding that address)

File: patches/tradeSequence.py [hunk 2]
```diff
@@ -267,14 +267,15 @@ def patchMermaid(rom):
...
         ld   a, [wTradeSequenceItem2]
         and  $10 ; scale
         ret  z
         ldh  a, [$FFF8]
-        and  $20
+        and  $20 ; ROOM_STATUS_EVENT_2
         ret  nz
     """), fill_nop=True)
+    rom.patch(0x18, 0x0971, 0x097A, ASM("ld hl, wTradeSequenceItem2\nres 4, [hl]"), fill_nop=True)  # Take the trade item
```
Comment (daid) [inline patches/tradeSequence.py:278]: Why make a 2nd patch when you could append this to the previous one?

**Review by daid — CHANGES_REQUESTED** [review]

File: patches/tradeSequence.py (hunk 2 above)
Comment (EpochFlame) [reply patches/tradeSequence.py:278]: they were separate in the other patches, no reason really. will merge with previous

File: patches/tradeSequence.py (hunk 1 above)
Comment (EpochFlame) [reply patches/tradeSequence.py:270]: ok, merging the patch with the adjacent patch should resolve this concern, but looking at the generated rom, the NOP from the earlier patch started at 0971. will merge the patches for cleanliness.

**Pull request merged**